### ISSUE TITLE
Animal data

### ISSUE THREAD

**Issue by aarcury-quandt** [midas-network/midas-data#4]
I don't think there's anything in our definitions that would preclude animal data from being included there, but it may be good to have an indicator that the data does pertain to animals.  There are 3 main types that I can think of

1. Data from animal illnesses - for example, PDPR only affects small ruminants. Data would probably be surveillance or culling data. This would be captured in the pathogen.
2. Data from zoonotic illnesses affecting animals - This could be something like yellow fever epizootics. 
3. Surveillance/data from animals where human health is the primary concern. 

- West Nile is zoonotic, but mosquito pools are generally used to aid disease prevention in humans, not birds. 
- Vector surveillance - ticks for lyme or tick borne encephalitis, raccoons for rabies, etc
- Animals as indicators without being infected or being a vectot. Testing dog feces for Trypanasoma parasites: dogs can't be infected, but the do eat the vector. I'm sure there are other examples.

Not categorized because I don't know where something like this would go: brucellosis (or similar) data in livestock. Human health AND livestock health are both of concern here. I'm also conflicted about avian (and other animal) influenzas. There are concerns about livestock health and wild animal die offs, but there's also concern about strains making the leap to infecting humans.

The other consideration is that maybe some data (e.g., category 3)  _don't_ need to be labeled as veterinary/animal data because human health is the primary focus.

**Comment by aarcury-quandt** [comment]
A fourth category would be from animal laboratory studies.

**Comment by hoganwr** [comment]
I see a request to add 4 topics, with beginnings of descriptions (I'd shy away from saying what we're doing here is definitions, otherwise if they're left up to me, they're going to end up as highly ontologically formal, rigorous, and precise definitions, and that is explicitly what you said you didn't want).  Probably would just add animal as a topic, and these four underneath it.  Is that sufficient?

**Comment by harryhoch** [comment]
@hoganwr , yes, I think this would be sufficient, but we should discuss.

**Comment by hoganwr** [comment]
I propose 4 new classes:

Class 1
label: animal disease data
parent class: topic
definition: surveillance data for the purposes of tracking disease in animal populations

Class 2
label: zoonotic diseases in animal data
parent class: animal disease data
definition: animal disease data about zoonotic diseases occurring in animals

Class 3
label: animal disease data of human health relevance
parent class: animal disease data
definition: animal disease data where the diseases tracked are of relevance to human health
comment: how does this differ from zoonotic disease?  Are there animal diseases of relevance to human health that are not zoonotic?

Class 4
label: animal laboratory studies
parent class: ?
definition: ?
comment: what is meant here?  Are you talking about basic science research as is routinely carried out on mice and rats for example?  Or is this clinical laboratory testing but as applied to animals (vs. huamns)?

**Comment by harryhoch** [comment]
@hoganwr , this looks good. @aarcury-quandt , how do the first 3 sound? Can you clarify the fourth? thanks.

**Comment by aarcury-quandt** [comment]
Class 1 and 2 are good.  Most everything will fit in there.

Class 3: When put this way, I would say something like avian influenza fits here.  We know about it, and we track it in animals because of the concern of it leaping to humans.  At the same time, I can only think of avian influenza for this class.

Class 4:  I can't think of anything for this. Vector surveillance  is the only thing I mentioned above that needs a home, but I wouldn't call it "animal laboratory studies".   Would I even call it "animal disease data"?  Not all vectors carry zoonotic diseases.

Slight tangent if we want to consider "vector surveillance", which could have 2 meanings: 1 Surveillance of the vectors themselves to monitor population size or detect new vectors in a region OR 2 Capturing vectors and testing them for diseases that affect humans to monitor the burden in the vector population. Both are relevant for disease modeling, and I imagine datasets for both are/will at some point become available

**Comment by hoganwr** [comment]
What specifically shall we do then?

I propose classes #1 - #3 as I wrote them.  Get rid of #4.

Do you also need two more classes for the vectors as you've described
them?  I can make proposals for those if so.

**Comment by aarcury-quandt** [comment]
I agree with the portal re: classes. I think we should have something for vectors. @harryhoch?

**Comment by harryhoch** [comment]
that's fine with me if need something for vectors.

**Comment by harryhoch** [comment]
Closing this issue as it has not risen to the level of importance.
